dev_env/docker/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV CHROME_VERSION "google-chrome-stable"
 RUN sed -i -- 's&deb http://deb.debian.org/debian jessie-updates main&#deb http://deb.debian.org/debian jessie-updates main&g' /etc/apt/sources.list \
   && apt-get update && apt-get install wget -y
 ENV CHROME_VERSION "google-chrome-stable"
-RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
```

Comment:
Was this intentional?

Reply:
Yes, this was failing in my local environment so I commented it out and then put it back in. This is related to the chrome question I asked in slack.